scientific/Portrait_Plot/ReusablePortraitPlot.html

```diff
@@ -11743,13 +13085,13 @@
 </style>
 
 <!-- Custom stylesheet, it must be in the same directory as the html file -->
-<link rel="stylesheet" href="custom.css">
+<link rel="stylesheet" href="file:///home/nadeau1/Downloads/custom.css">
```

Comment:
In addition to using a local copy of the custom.css file, it looks like this version of the custom.css file is noticeably different from the old version. (This phenomena shows up in many if not all the html files I've looked at so far.)